src/controllers/MonitorController.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Cronbeat\Controllers;
+
+use Cronbeat\RedirectException;
+
+class MonitorController extends BaseController {
```

Comment:
@jetbrains-junie cover with tests (like DashboardControllerTest.php). ensure also monitors with multiple pages can be served.